### PR TITLE
@kanaabe: Random cleanups

### PR DESCRIPTION
**Pull request by craigspaeth**
Sorry for the scatter-brain PR—the diff is more intimidating than the actual changes here.

- Dropped artsy.net/gallery-insights/opt-in and artsy.net/social-media-toolkit now that we built these pages in Marketo
- Killed off Mailchimp code
- Namespaced the classes and ids of elements in apps/partner2 to fix the weird issues reported in #partner-s-help

### CODE REVIEW

File: apps/articles/index.coffee [hunk 1]
```diff
@@ -22,3 +22,4 @@ app.get '/articles', routes.articles
 app.get sd.TEAM_BLOGS, routes.teamChannel
 app.get '/venice-biennale-2015', routes.section
 app.post '/editorial-signup/form', routes.editorialForm
+app.get '/gallery-insights/opt-in', (_, res) -> res.redirect '/gallery-insights'
```
Comment (kanaabe) [inline apps/articles/index.coffee]: `/gallery-insights` doesn't currently have a signup, is that fine? Just the individual article pages do right now.

File: apps/mailchimp_subscribe/routes.coffee [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-      id: req.body.listId
-      email: email: req.body.email
-      merge_vars: req.body.mergeVars
-      double_optin: false
-      send_welcome: false
-    ).end (err, response) ->
-      if (response.ok)
-        res.send req.body
-      else
-        res.send(response.status, error: response.body.error)
```
Comment (kanaabe) [inline apps/mailchimp_subscribe/routes.coffee]: Nice! Glad this is all going away.

File: apps/articles/index.coffee (hunk 1 above)
Comment (craigspaeth) [inline apps/articles/index.coffee]: Yep yep I checked with @jessekedy on this.

**Comment by kanaabe** [comment]
Cool! This looks all good to me after tests are 💚 . For posterity, what are the links to the Marketo pages replacing /optin and /social-media-toolkit?

**Comment by craigspaeth** [comment]
Toolkit: http://insights.artsy.net/Social-Media-Toolkit
Opt-in: I believe we're just not using this landing page anymore and linking directly to the articles

Thanks for the review, I'll merge when I get it 💚

**Pull request merged**